classes/mender-install.bbclass

```diff
@@ -1,6 +1,10 @@
 PREFERRED_VERSION_go_cross = "1.6%"
 
-#Add support for systemd
+# Add support for systemd
+#
+# Note that a very common error with systemd is that the system hangs during the
+# boot process while looking for devices. This is almost always because the
+# kernel feature CONFIG_FHANDLE is not enabled.
 DISTRO_FEATURES_append = " systemd"
```

Comment:
Firstly, our integration is limited to shipping `mender.service`, so that's not a big deal. We may equally well ship a simple sysvinit script and work with poky.

Try to follow the principle of least surprise. Integration with systemd is a rather big deal, and people end up running into different problems, even delaying/dropping the work. Now, if one ends up evaluating mender with an existing non-systemd distribution (or even poky), things may blow up. Tracking the that back to `mender-install.bbclass` will be surprising to say the least.